internal/cmd/plugin/command_install.go

```diff
@@ -0,0 +1,175 @@
+package plugin
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/go-yaml/yaml"
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/types"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+func (c *command) newInstallCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install",
+		Short: "Install Confluent CLI plugins.",
+		Long:  "Install available Confluent CLI plugins from the confluentinc/cli-plugins repository.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.install,
+	}
+
+	cmd.Flags().String("plugin-directory", "", "The plugin installation directory; this must be a directory in your $PATH. If not specified, a default will be selected based on your OS.")
```

Comment:
```suggestion
	cmd.Flags().String("plugin-directory", "", "The plugin installation directory. This must be a directory in your $PATH. If not specified, a default will be selected based on your OS.")
```

Reply:
Can we tell the user what the default is? I assume /usr/local/bin on macOS?

Reply:
Yeah it's `/usr/local/bin` on unix systems and the CLI installation directory on Windows.

That's what I ultimately went with because the parts of the path common to all users is the system root and some windows32 folders, none of which I want to write to. I guess there's no real guarantee that the user will actually have added the CLI install directory to their path, but I issue a warning if the plugin directory isn't in the path.

I'll specify that in the flag description.